src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.MaxResponseHeadersLength.cs

```diff
@@ -45,16 +45,30 @@ public void ValidValue_SetGet_Roundtrips(int validValue)
         }
 
         [Fact]
-        [OuterLoop("Uses external server")]
         public async Task SetAfterUse_Throws()
         {
-            using (HttpClientHandler handler = CreateHttpClientHandler())
-            using (HttpClient client = CreateHttpClient(handler))
+            await LoopbackServer.CreateClientAndServerAsync(async uri =>
```

Comment:
As long as we're rewriting tests, can we write it in a way that would work for both HTTP/1.1 and HTTP/2.0, e.g. use GenericLoopbackServer?

Reply:
Done.